scripts/etc/init.d/rcS

```diff
@@ -1115,6 +1121,15 @@ if [ -e /bootfs/post-install.txt ]; then
     echo "================================================="
 fi
 
+# modify installed network settings
+if [ -f /rootfs/etc/wpa_supplicant/wpa_supplicant.conf ]; then
+    if [ $(grep "iface" /rootfs/etc/network/interfaces |grep -v "lo\|eth0" |wc -l) -ne 0 ]; then
```

Comment:
please add spaces after the pipe characters (the actual pipes, not the character in the regex 😉 )